deploy/examples/create-external-cluster-resources.py

```diff
@@ -825,12 +831,20 @@ def get_rbd_provisioner_caps_and_entity(self):
         }
         if self._arg_parser.restricted_auth_permission:
             rbd_pool_name = self._arg_parser.rbd_data_pool_name
+            alias_rbd_pool_name = self._arg_parser.alias_rbd_data_pool_name
             cluster_name = self._arg_parser.cluster_name
             if rbd_pool_name == "" or cluster_name == "":
                 raise ExecutionFailureException(
                     "mandatory flags not found, please set the '--rbd-data-pool-name', '--cluster-name' flags"
                 )
-            entity = f"{entity}-{cluster_name}-{rbd_pool_name}"
+            if rbd_pool_name.count(".") != 0:
```

Comment:
Instead of checking if pool name has the `.`, how about we check for the `.` and `_` in the alias name? Because if they specify an alias with those chars we know it will fail.